pimd/pim6_cmd.c

```diff
@@ -48,9 +48,97 @@ static struct cmd_node debug_node = {
 	.config_write = pim_debug_config_write,
 };
 
+DEFUN (interface_ipv6_mld_query_max_response_time,
+       interface_ipv6_mld_query_max_response_time_cmd,
+       "ipv6 mld query-max-response-time (1-65535)",
+       IPV6_STR
+       IFACE_MLD_STR
+       IFACE_MLD_QUERY_MAX_RESPONSE_TIME_STR
+       "Query response value in deci-seconds\n")
+{
+    const struct lyd_node *pim_enable_dnode;
+    pim_enable_dnode =
+        yang_dnode_getf(vty->candidate_config->dnode,
+                "%s/frr-pim:pim/pim-enable", VTY_CURR_XPATH);
+    if (!pim_enable_dnode) {
+        nb_cli_enqueue_change(vty, "./enable", NB_OP_MODIFY,
+                      "true");
+    } else {
+        if (!yang_dnode_get_bool(pim_enable_dnode, "."))
+            nb_cli_enqueue_change(vty, "./enable",
+                          NB_OP_MODIFY, "true");
+    }
+    nb_cli_enqueue_change(vty, "./query-max-response-time", NB_OP_MODIFY,
+                  argv[3]->arg);
+    return nb_cli_apply_changes(vty, FRR_GMP_INTERFACE_XPATH,
+                    "frr-routing:ipv6");
+}
+DEFUN (interface_no_ipv6_mld_query_max_response_time,
+       interface_no_ipv6_mld_query_max_response_time_cmd,
+       "no ipv6 mld query-max-response-time [(1-65535)]",
+       NO_STR
+       IPV6_STR
+       IFACE_MLD_STR
+       IFACE_MLD_QUERY_MAX_RESPONSE_TIME_STR
+       IGNORED_IN_NO_STR)
+{
+    nb_cli_enqueue_change(vty, "./query-max-response-time", NB_OP_DESTROY,
+                  NULL);
+    return nb_cli_apply_changes(vty, FRR_GMP_INTERFACE_XPATH,
+                    "frr-routing:ipv6");
+}
+DEFUN_HIDDEN (interface_ipv6_mld_query_max_response_time_dsec,
+          interface_ipv6_mld_query_max_response_time_dsec_cmd,
+          "ipv6 mld query-max-response-time-dsec (1-65535)",
```

Comment:
Do we have this command in Cisco? HIDDEN commands are required to be added only when we need backward compatibility.